crypto/rand/randfile.c

```diff
@@ -104,6 +104,12 @@ static __FILE_ptr32 (*const vms_fopen)(const char *, const char *, ...) =
 
 #define RFILE ".rnd"
 
+#ifdef EINTR
+# define INTERRUPTED(in) (ferror(in) && errno == EINTR)
+#else
+# define INTERRUPTED (0)
```

Comment:
Right! I must have missed the white space and tested `#define FOO(0)`. But macro definition is wrong either way. I mean definition without `(x)` is bad too, as it gets preprocessed as `(0)(in)`...